h/api/presenters.py

```diff
@@ -90,14 +93,41 @@ def asdict(self):
 
     @property
     def permissions(self):
-        read = self.annotation.userid
-        if self.annotation.shared:
-            read = 'group:{}'.format(self.annotation.groupid)
+        return _permissions(self.annotation)
+
+
+class AnnotationSearchIndexPresenter(AnnotationBasePresenter):
```

Comment:
Some basic testing would be nice here, so we're not going to accidentally break search indexing.